config/settings.py

```diff
@@ -15,5 +15,5 @@ class Settings(BaseSettings):
     output_file: str = "data/output.csv"
 
     class Config:
-        env_file = "config/.env.example"
+        env_file = "config/.env"
```

Comment:
I think by default we should leave `.example` and only override in `main.py` because default is used in tests and `.env` file might not even exist in test environments